dev/tests/static/testsuite/Magento/Test/Integrity/_files/dependency_test/whitelist/redundant_dependencies_webapi.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+return [
+    'Magento\GraphQl' => [
+        'Magento\Webapi' => 'Magento\Webapi'
+    ]
+];
```

Comment:
Hi @sasha19957099 @danielrenaud,
Could you explain why this file was added? Seems like test shouldn't fail anymore as it has declared dependency on web api module. An I wrong?